### DISCUSSION TITLE
survey functionality for projects that track by transect, not just by server unit

### DISCUSSION THREAD

**Discussion by urapadmin** [arch-kiosk/arch-kiosk-office#1240] [users: ideas for new features]
Moved here from "ideas and drafts for the recording system" in the wiki, originally written by @lbestock. 

Nuri has worked on such a project, and spoken with team members to reconstruct what was done. This is something we envisioned needing to support, and here's a spot for Nuri to record his recollections:

When I worked on a survey project we would divide our survey area (usually a field) into straight line transects that we tried to have cut through different parts of the area such as elevation, plant life or visible features. This means that when we looked back at the data it was possible to see if for instance the different elevations changed the quantity or type of finds along the transect. One person then walks down each transect and stops every few meters (depending on the length of the field and how full of material it is) to document everything they see within around a square meter of where they are standing. The documentation of material is split into each stop along the transect rather than just showing how much was found along the whole thing. This method of survey is able to more accurately say where the most interesting areas are within the survey unit.

The current transect recording in filemaker could be be used to accomadate this method quite easily I think if every transect row being filled in was instead each stop along the transect. I could draw up how I was thinking it could be used if that would be helpful.

It might also be interesting to give each transect its own space where you can describe the different elevations, plant life and visible features. This could be similar to the descriptions we give to features and contexts.

**Comment by luizaogs** [comment]
A totally insane thought, but if Oren and I start our project in Egypt we will be doing some surveying, so I've been thinking about surveying. It would be cool to have a little Kiosk phone app that works as a counter and that would send sherd counts (or what have you) directly to the recording app by transect.

A daydream that I might bother you about in future if we get permission to do anything at the site at all. Writing it here now so I don't completely forget about it.
